### PR TITLE
swapped out the species IDs for CamelCase

### PR DESCRIPTION
**Pull request by andrewkern**
hey @gtsambos! 

wondering if you might review this PR from me which is aimed at closing https://github.com/popgensims/stdpopsim/issues/318

### CODE REVIEW

**Comment by andrewkern** [comment]
ack forgot to update the tests... hold on a minute...

**Comment by gtsambos** [comment]
Yup! Lmk when the tests have passed :)

**Comment by andrewkern** [comment]
okay ready for you @gtsambos! thx

File: stdpopsim/species.py [hunk 1]
```diff
@@ -61,10 +61,11 @@ class Species(object):
     Class representing a species in the catalog.
 
     :ivar id: The unique identifier for this species. The species ID is
-        usually an abbreviation of the species name, which does not
-        contain any spaces or puncutation.. The usual scheme is to
+        the three first letters of the genus name followed by the first
+        three letters of the species name, and does not
+        contain any spaces or puncutation. The usual scheme is to
```
Comment (gtsambos) [inline stdpopsim/species.py]: small type - 'punctuation'

Comment (gtsambos) [reply stdpopsim/species.py]: lol 'typo'

Comment (andrewkern) [reply stdpopsim/species.py]: oh it's a good one too!

**Comment by andrewkern** [comment]
i'll rename those pongo files, fix that type, rebase, and merge if that's okay by you @gtsambos

**Comment by gtsambos** [comment]
a-okay by me!

**Pull request merged**